### PR TITLE
build: header desktop

### PR DESCRIPTION
**Pull request by ossset**

### CODE REVIEW

File: css/header.css [hunk 1]
```diff
@@ -53,4 +53,37 @@
...
+        font-size: 13px;
+        text-transform: uppercase;
+        font-family: "Circe Regular";
+        color: #2d2829;
+    }
+    .header__link:hover{
+        color: #f61f5e;
+    }
+    .header__phone{
+        margin-left: 10%;
```
Comment (uladzislau-kirychenka) [inline css/header.css]: Please avoid using percantage for margin, padding

File: css/individual.css [hunk 2]
```diff
@@ -533,7 +536,140 @@
...
+        font-size: 22px;
+    }
+
+    .individual__card--inner::after {
+        font-size: 18px;
+        top: 500px;
+        right: 120px;
+    }
+
+    .individual__card--size {
```
Comment (uladzislau-kirychenka) [inline css/individual.css]: Unclear naming

File: css/individual.css [hunk 3]
```diff
@@ -533,7 +536,140 @@
...
+    .individual__slider-item--masrhmello {
+        background: url(../assets/img/marsh-lg.png)
+    }
+
+    .individual__card--last {
+        justify-content: flex-end;
+        padding-right: 70px;
+    }
+
+    .individual__card--last * {
```
Comment (uladzislau-kirychenka) [inline css/individual.css]: avoid * selector

File: css/main.css [hunk 4]
```diff
@@ -133,4 +135,9 @@ ul {
     .slider-button{
         display: block;
     }
+}
+@media (min-width:1080px){
+    .container{
+        width: 89%;
```
Comment (uladzislau-kirychenka) [inline css/main.css:138]: is it possible not to use percentage?

File: css/pros.css [hunk 5]
```diff
@@ -107,4 +107,43 @@
...
+        width: 400px;
+    }
+    .pros__item{
+        display: flex;
+        align-items: center;
+    }
+    .pros__item-wrapper{
+        padding-left: 30px;
+    }
+    .pros__item-wrapper--ingridients{
```
Comment (uladzislau-kirychenka) [inline css/pros.css:146]: Typo in ingridients

File: index.html [hunk 6]
```diff
@@ -166,18 +168,19 @@ <h2 class="individual__title">
                 Соберите торт в 3 шага
             </p>
             <div class="individual__card individual__card--inner">
-                <div class="individual__card-title">
+                <!-- <div class="individual__card-title"> -->
```
Comment (uladzislau-kirychenka) [inline index.html]: remove useless comments

File: index.html [hunk 7]
```diff
@@ -166,18 +168,19 @@ <h2 class="individual__title">
...
                 <div class="slider-button slider-button--left">
                 </div>
                 <div>
                     <picture>
                         <source srcset="./assets/img/inner-sm.png" class="individual__img" media="(max-width: 769px)">
-                        <img src="./assets/img/inner-md.png" alt="inner" class="individual__img individual__img--md">
+                        <source media="(max-width :1080px)" class="individual__img" srcset="./assets/img/inner-md.png">
+                        <img src="./assets/img/inner-lg.png" alt="inner" class="individual__img individual__img--md">
                     </picture>
                     <!-- <img class="individual__img" src="./assets/img/inner-sm.png" alt="inner" />
```
Comment (uladzislau-kirychenka) [inline index.html]: remove useless comments

File: index.html [hunk 8]
```diff
@@ -207,21 +210,21 @@ <h2 class="individual__title">
...
+                <!-- </div> -->
                 <div class="individual__name individual__name--circle individual__card--size">
                     <div class="individual__item">
                         <div class=" circle-card">
                             <picture>
-                                <source media="(max-width: 769px)" srcset="./assets/img/circle-sm.png"
-                                    class="individual__img  circle-img">
+                                <source media="(max-width: 769px)" srcset="./assets/img/circle-sm.png" class="individual__img  circle-img">
+                                <source media="(max-width: 1080px)" srcset="./assets/img/circle1-md.png">
                                 <!-- <img class="individual__img  circle-img" src="./assets/img/circle-sm.png" alt="inner" /> -->
```
Comment (uladzislau-kirychenka) [inline index.html]: remove useless comments

File: index.html [hunk 9]
```diff
@@ -270,12 +281,12 @@ <h2 class="individual__title">
                 </div>
             </div>
             <div class="individual__card individual__card--last">
-                <div class="individual__card-title individual__card-title--last">
+                <!-- <div class="individual__card-title individual__card-title--last"> -->
```
Comment (uladzislau-kirychenka) [inline index.html]: remove useless comments

File: index.html [hunk 10]
```diff
@@ -491,7 +507,7 @@ <h2 class="pros__title">
         <div class="pros__items-container">
             <div class="pros__item pros__item--ingidients">
                 <img src="./assets/img/ingridients-iconpros-sm.png" alt="natural" />
-                <div class="pros__item-wrapper">
+                <div class="pros__item-wrapper pros__item-wrapper--ingridients">
```
Comment (uladzislau-kirychenka) [inline index.html:507]: typo

**Review by uladzislau-kirychenka — CHANGES_REQUESTED** [review]
See comments above

File: css/main.css (hunk 4 above)
Comment (ossset) [reply css/main.css:138]: I'm using percentage here for a more adaptive look of the 770px-1037px layout.

File: css/header.css (hunk 1 above)
Comment (ossset) [reply css/header.css]: done

File: css/individual.css (hunk 3 above)
Comment (ossset) [reply css/individual.css]: ```done```

File: index.html (hunk 6 above)
Comment (ossset) [reply index.html]: ```done```

File: index.html (hunk 7 above)
Comment (ossset) [reply index.html]: done

File: index.html (hunk 8 above)
Comment (ossset) [reply index.html]: done

File: index.html (hunk 9 above)
Comment (ossset) [reply index.html]: done

File: index.html (hunk 10 above)
Comment (ossset) [reply index.html:507]: done

File: css/pros.css (hunk 5 above)
Comment (ossset) [reply css/pros.css:146]: done

File: css/individual.css (hunk 2 above)
Comment (ossset) [reply css/individual.css]: done

File: css/individual.css [hunk 11]
```diff
@@ -533,7 +536,135 @@
...
+
+    .individual__slider-item--cherry {
+        background: url(../assets/img/berries-lg.png)
+    }
+
+    .individual__slider-item--design {
+        padding-top: 333px;
+    }
+
+    .individual__slider-item--masrhmello {
```
Comment (uladzislau-kirychenka) [inline css/individual.css]: Typo

File: css/individual.css [hunk 12]
```diff
@@ -533,7 +536,135 @@
...
+    .individual__card--last {
+        justify-content: flex-end;
+        padding-right: 70px;
+    }
+
+    .individual__card--last .slider-button--left, .slider-button--right, .individual__slider-item {
+        margin-left: 40px;
+    }
+
+    .individual__card--last .slider-button {
```
Comment (uladzislau-kirychenka) [inline css/individual.css]: Can you use separate element instead of this selector

File: css/pros.css [hunk 13]
```diff
@@ -107,4 +107,43 @@
...
+        font-size: 35px;
+    }
+    .pros__subtitle::after{
+        background: url(../assets/img/berries-pros-lg.png) no-repeat;
+        width: 555px;
+        height: 690px;
+    }
+    .pros__items-container{
+        height: 450px;
+        padding-right: 20%;
```
Comment (uladzislau-kirychenka) [inline css/pros.css]: Do you need this percentage padding?

**Review by uladzislau-kirychenka — CHANGES_REQUESTED** [review]

File: css/individual.css (hunk 11 above)
Comment (ossset) [reply css/individual.css]: done

File: css/individual.css (hunk 12 above)
Comment (ossset) [reply css/individual.css]: done

File: css/pros.css (hunk 13 above)
Comment (ossset) [reply css/pros.css]: done

**Pull request merged**